sample/Sample.cs

```diff
@@ -123,13 +139,20 @@
     await Task.Delay(TimeSpan.FromMinutes(2));
 }
 
-static void LogTokenEvents(ConfigurationOptions configurationOptions)
+static void LogTokenEvents(ConfigurationOptions configurationOptions, string option)
 {
     if (configurationOptions.Defaults is IAzureCacheTokenEvents tokenEvents)
     {
         static void Log(string message) => WriteLine($"{DateTime.Now:s}: {message}");
+        if (option.Equals("5"))
+        {
+            tokenEvents.AccessTokenRefreshed += (sender, accessToken) => Log($"Token refreshed. New token will expire at {accessToken.ExpiresOn}");
+        }
+        else
+        {
+            tokenEvents.TokenRefreshed += (sender, authenticationResult) => Log($"Token refreshed. New token will expire at {authenticationResult.ExpiresOn}");
+        }
```

Comment:
We can probably extract out the "option 5" logic from this method since it's only useful for option 5.  Let me think of a different way to structure the code, and I'll write something back.

Reply:
Perhaps we can make it something like a boolean with default `false` call it like "isUsingAccessToken" or something similar.

Also in this lambda expression do we need to provide `sender` as an input? Is it being used anywhere?

Reply:
I believe this could be used in a client app where its more important to know the sender but since its just the sample app, we don't use it explicitly. 
CC: @philon-msft for more context